### PR TITLE
add novaxr output file

### PR DESCRIPTION
**Pull request by Andrey1994**
Signed-off-by: Andrey Parfenov <a1994ndrey@gmail.com>

### CODE REVIEW

File: OpenBCI_GUI/BoardNovaXR.pde [hunk 1]
```diff
@@ -3,6 +3,7 @@ import brainflow.*;
 import org.apache.commons.lang3.ArrayUtils;
 
 final boolean novaXREnabled = false;
+final boolean usingNovaXREmulator = false;
```
Comment (retiutut) [inline OpenBCI_GUI/BoardNovaXR.pde:6]: Added this flag to help with development

**Review by retiutut — APPROVED** [review]
Control Panel SessionDataBox and NovaXRBox have been refactored and now use all cp5 UI objects. Should work as expected, though aesthetic changes may be necessary once team has tested the change.

File: OpenBCI_GUI/BoardNovaXR.pde (hunk 1 above)
Comment (Andrey1994) [reply OpenBCI_GUI/BoardNovaXR.pde:6]: is it better than adding one line for ip address and comment it out?

**Comment by Andrey1994** [comment]
looks fine for me

File: OpenBCI_GUI/BoardNovaXR.pde (hunk 1 above)
Comment (retiutut) [reply OpenBCI_GUI/BoardNovaXR.pde:6]: It helps a little, also since it is next to this other boolean.

**Pull request merged**